phalcon/mvc/model/criteriainterface.zep

```diff
@@ -192,6 +193,6 @@ interface CriteriaInterface
 	/**
 	 * Executes a find using the parameters built with the criteria
 	 */
-	public function execute() -> <ResultsetInterface>;
+	public function execute(<TransactionInterface> transaction = null) -> <ResultsetInterface>;
```

Comment:
This change will break BC if someone implements `Phalcon\Mvc\Model\CriteriaInterface`.